Samples/auth/Outlook-Add-in-SSO-NAA-Identity/README.md

```diff
@@ -0,0 +1,249 @@
+
+1. The Outlook add-in (SPA client) uses the MSAL.js library to obtain an access token from Microsoft Entra ID. The MSAL.js library returns an id token, access token, and refresh token.
+1. The access token is passed in a web request to a protected web API service. The web request validates the access token and confirms the user has permissions to perform the action in the API. The access token contains any identity claims the web API needs.
+
+![A diagram showing the Outlook add-in getting an access token from Microsoft Entra ID and calling the protected web API with the access token.](architecture-diagram.png)
+
+
+
+## Applies to
+
+- Outlook on Windows (new and classic), Mac, mobile, and on the web.
+
+## Prerequisites
+
+- Office connected to a Microsoft 365 subscription (including Office on the web).
+- [Node.js](https://nodejs.org/) version 16 or greater.
+- [npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm) version 8 or greater.
+
+## Register the sample applications with your Microsoft Entra tenant
+
+### Register the web API service
+
+The web API service is a Node.js server that requires an application registration to authorize calls to the todo list database.
+
+1. Go to the [Azure portal - App registrations](https://go.microsoft.com/fwlink/?linkid=2083908) page to register your app.
+1. Sign in with the **_admin_** credentials to your Microsoft 365 tenancy. For example, **MyName@contoso.onmicrosoft.com**.
+1. Select **New registration**. On the **Register an application** page, set the values as follows.
+
+   - Set **Name** to `Contoso-Web-API-Server`.
+   - Set **Supported account types** to **Accounts in this organizational directory only**.
+   - Select **Register**.
+
+1. In the app's registration screen, select the **Expose an API** blade to the left to open the page where you can publish the permission as an API for which client applications can obtain access tokens for. The first thing that we need to do is to declare the unique resource URI that the clients will be using to obtain access tokens for this API. To declare an resource URI(Application ID URI), follow the following steps:
+
+    - Select **Add** next to the **Application ID URI** to generate a URI that is unique for this app.
+    - For this sample, accept the proposed **Application ID URI** (`api://{clientId}`) by selecting **Save**.
+
+    Read more about Application ID URI at [Validation differences by supported account types (signInAudience)](https://docs.microsoft.com/azure/active-directory/develop/supported-accounts-validation).
+
+### Publish delegated permissions
+
+All APIs must publish a minimum of one scope, also called Delegated Permission, for the client apps to obtain an access token for a user successfully. To publish a scope, follow these steps:
+
+1. Select **Add a scope** button open the **Add a scope** screen and Enter the values as indicated below:
```

Comment:
This list and the next one have different numbering styles. Let's pick one to be consistent.

Also, consider making the "Repeat" step, Step 2. It'll make it more obvious that's an action, instead of informatino text.